frontend/src/app/Settings/Users/ManageUsers.tsx

```diff
@@ -188,52 +251,54 @@ const ManageUsers: React.FC<Props> = ({
               users={usersState}
               onChangeActiveUser={onChangeActiveUser}
             />
-            <div className="tablet:grid-col">
-              <div className="user-header">
-                <h2 className="display-inline-block margin-top-2 margin-bottom-105">
-                  {displayFullName(
+            <div className="tablet:grid-col padding-left-2">
+              <div className="user-header grid-row flex-row flex-align-center">
+                <h2 className="display-inline-block margin-y-1">
+                  {displayFullNameInOrder(
                     activeUser.firstName,
                     activeUser.middleName,
                     activeUser.lastName
                   )}
                 </h2>
                 {activeUser.id === loggedInUser.id ? (
-                  <span className="usa-tag margin-left-1">YOU</span>
+                  <span className="usa-tag margin-left-1 bg-base-lighter text-ink">
+                    YOU
+                  </span>
                 ) : null}
               </div>
               <div className="user-content">
-                <p>
-                  Permissions to manage settings and users are limited to admins
-                  only
+                <p className="text-base">
+                  Admins have full access to conduct tests, manage results and
+                  profiles, and manage settings and users
                 </p>
                 <UserRoleSettingsForm
                   activeUser={activeUser}
                   loggedInUser={loggedInUser}
                   onUpdateUser={updateUser}
                 />
 
-                {process.env.REACT_APP_USER_FACILITIES_ENABLED === "true" ? (
+                {process.env.REACT_APP_VIEW_USER_FACILITIES === "true" ? (
                   <UserFacilitiesSettingsForm
                     activeUser={activeUser}
                     allFacilities={allFacilities}
                     onUpdateUser={updateUser}
                   />
                 ) : null}
               </div>
-              <div className="usa-card__footer display-flex flex-justify margin-top-5">
+              <div className="usa-card__footer display-flex flex-justify margin-top-5 padding-x-0">
```

Comment:
We're using the `__footer` class here but not the `__header`, wondering if that is intentional?